Library/Formula/ltsa.rb

```diff
@@ -0,0 +1,19 @@
+class Ltsa < Formula
+  desc "LTS Analyser"
+  homepage "http://www.doc.ic.ac.uk/~jnm/book/ltsa/LTSA_applet.html"
+  url "http://www.doc.ic.ac.uk/~jnm/book/ltsa/ltsatool.zip"
```

Comment:
I don't maintain the package or website. Is there no way of getting it brew-able (beyond privately) without one?